newrelic/api/log.py

```diff
@@ -88,4 +89,6 @@ def format(self, record):
 
         if record.exc_info:
             output.update(format_exc_info(record.exc_info))
-        return json.dumps(output)
+
+    def format(self, record):
+        return json.dumps(self.log_record_to_dict(record))
```

Comment:
```suggestion
        return output

    def format(self, record):
        return json.dumps(self.log_record_to_dict(record))
```

You're missing a return statement on `log_record_to_dict` 🙂 

Other than that, LGTM.

Reply:
How embarrassing 😊 

Fixed.